cavern/src/intTest/java/org/opencadc/cavern/TransferTest.java

```diff
@@ -173,4 +183,108 @@ public void testPreauthToken() {
             Assert.fail("Unexpected error: " + e);
         }
     }
+
+    /**
+     * Test that a user can update a data node they do not own, but have write permission to the node.
+     *
+     * The authSubject user is the owner of the parent ContainerNode and child DataNode.
+     * The altSubject user is a member of the altGroup group.
+     * The altGroup by default does not have write access to the parent or child nodes.
+     *
+     * The altGroup group is given write access to the child node, which should
+     * allow the altSubject to write to the child node.
+     */
+    @Test
```

Comment:
ok, so this method is here but the "enable" is in the test suite? I assume the rework is incomplete...